tpm2/encoding_test.go

```diff
@@ -385,6 +385,19 @@ func TestECCParamsEncodeDecode(t *testing.T) {
 		t.Fatalf("decodeECCParams: %v", err)
 	}
 
+	if params.Point.X().Cmp(pk.PublicKey.X) != 0 {
+		t.Fatalf("got: %+v\nwant: %+v", params.Point.X(), pk.PublicKey.X)
```

Comment:
Clarify what got/want in these failure messages are

Reply:
Sorry, I meant in the actual `t.Fatalf` text, so you don't need to look at test code to understand which check failed.

Reply:
np, added